python/tests/types/test_numpy.py

```diff
@@ -70,4 +70,4 @@ def resize_mask(arr: np.ndarray) -> np.ndarray:
     df = df.withColumn("resized", resize_mask("data"))
     df.show()
 
-    df.write.format("rikai").save(str(tmp_path))
+    df.write.format("rikai").mode("overwrite").save(str(tmp_path))
```

Comment:
Can we use `df.write.format("rikai").save(str(tmp_path / "data"))` instead of specify overwrite manually in all files.